src/types/intentions.ts

```diff
@@ -1,8 +1,17 @@
-const AVIARY_INTENTIONS = {
+const CORE_AVIARY_INTENTIONS = {
   primary: "primary",
   system: "system",
   danger: "danger",
   success: "success",
+};
+
+const AVIARY_INTENTIONS = {
+  ...CORE_AVIARY_INTENTIONS,
+  warning: "warning",
+};
+
+const AVIARY_BUTTON_INTENTIONS = {
+  ...CORE_AVIARY_INTENTIONS,
```

Comment:
not sure much can be done, but in `hwadmin`, we still have to exclude `success` from Button intentions

It will look like `intention?: Exclude<AviaryButtonIntentions, "success">;` which is just a little funny to read

Reply:
Hmm, true, I'll maybe then pluck and set that up here properly then